backend/.gitignore

```diff
@@ -2,6 +2,9 @@
 10_data.sql
 *.log
 
+/backend/node_modules/
```

Comment:
Since the backend is written in Go, I think the backend folder shouldn't have `node_modules` 🤔

Reply:
OMG THAT MAKES SO MUCH SENSE

Reply:
i think i added it by mistake a long time ago but never realized
thanks for pointing that out

Reply:
ill just delete it haha

Reply:
Yay! Problem solved!